src/app/core/components/Sidenav/Sidenav.tsx

```diff
@@ -97,26 +97,28 @@ const Sidenav = ({
   const isB2BWorkspace = !!useSelector(workspacesSelectors.getSelectedWorkspace);
   const isLoadingCredentials = useAppSelector((state: RootState) => state.workspaces.isLoadingCredentials);
   const pendingInvitations = useAppSelector((state: RootState) => state.shared.pendingInvitations);
+  const selectedWorkspace = useAppSelector(workspacesSelectors.getSelectedWorkspace);
+  const workspaceUuid = selectedWorkspace?.workspaceUser.workspaceId;
 
   const itemsNavigation: SideNavItemsProps[] = [
     {
-      to: '/',
+      to: workspaceUuid ? `/?workspaceid=${workspaceUuid}` : '/',
       isActive: isActiveButton('/') || isActiveButton('/file/:uuid') || isActiveButton('/folder/:uuid'),
       label: translate('sideNav.drive'),
       icon: FolderSimple,
       iconDataCy: 'sideNavDriveIcon',
       isVisible: true,
     },
     {
-      to: '/backups',
+      to: workspaceUuid ? `/backups/?workspaceid=${workspaceUuid}` : '/backups',
```

Comment:
backups and recents views are not in b2b. 
Maybe could create a function that generates the workspace url for a given path